examples/mbedTLS/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-mbedtls)
+
+hunter_add_package(mbedtls)
+find_library(
```

Comment:
Yeah, I thought about this but sadly the project does not support CMake config code generation yet.

Do you want me to hunterize this package inside `hunter_packages` and add CMake code to generate `mbedtlsConfig.cmake` ?